### PR TITLE
[Merged by Bors] - Generate olm bundles for openshift

### PR DESCRIPTION
**Pull request by adwk67**
# Description

This PR is used to intentionally bundle a specific version of the commons-operator (23.1.0) in order to test olm deployment with zookeeper/commons operators. The manifests are operator/version specific but the `makefile` change can be incorporated into operator-templating later on.

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: deploy/olm/23.1.0/bundle.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```
Comment (razvan) [inline deploy/olm/23.1.0/bundle.sh:1]: The script could be moved up a level and could be made version agnostic.

**Review by razvan — APPROVED** [review]
lgtm

File: deploy/olm/23.1.0/bundle.sh (hunk 1 above)
Comment (adwk67) [reply deploy/olm/23.1.0/bundle.sh]: Done.

**Comment by adwk67** [comment]
bors merge

**Pull request closed without merging**